pkg/testutil/prometheus.go

```diff
@@ -304,10 +320,15 @@ func createBlock(
 	series []labels.Labels,
 	numSamples int,
 	mint, maxt int64,
+	blockDelay time.Duration,
 	extLset labels.Labels,
 	resolution int64,
 	tombstones bool,
 ) (id ulid.ULID, err error) {
+	if err != nil {
+		return id, errors.Wrap(err, "create block id")
+	}
```

Comment:
Should this error be moved below Line 401, I guess?

Reply:
Can you check why the tests are breaking?